Update offence code in court-case-service testdata And FIx K8s Pod deletion logic

Obsiye opened the pull request:
- Update offence code in court-case-service testdata

This is because the current offence code can't be found when court-case-service makes an api call for it.

-- 

- Fix pod timeout threshold ... allow pod to take up to 30 seconds long so that it can have enough time to run its commands

marcus-bcl commented:
Thanks for this @Obsiye.  I've ran the court-case-and-delius test from your branch and it still fails, is that expected?  It was failing before, but just wanted to check.

The report is here: https://ministryofjustice.github.io/hmpps-probation-integration-e2e-tests/playwright-report/test/update-court-case-service-offence-code/2024-08-28/add07a7950f5e90b2d9a529cd42f52e5b2e73636/10597358842/1/#?testId=c4072ba92dee408e66de-00fd3e863f8e06465333

Obsiye committed:
Fix incorrect logic when shutting pod down
The logic should be:
If the pod has been running for more than 30 seconds, delete the pod.

However, at the moment, it's deleting / completing any pod that takes
less than 30 seconds which means it deletes the pod right after
it has started.

This change should allow the pod to run it's commands in time
